modules/data_release/php/data_release.class.inc

```diff
@@ -104,6 +104,12 @@ class Data_Release extends \NDB_Menu_Filter
         $this->tpl_data['superuser']        = $superuser;
         $this->tpl_data['userids']          = $userids;
         $this->tpl_data['data_release_ids'] = $data_release_ids;
+        if (isset($_GET['addpermissionSuccess'])
+            && $_GET['addpermissionSuccess'] == "false"
+        ) {
+            $this->tpl_data['err'] = $_GET['user']." already has the permisson for "
+                                     .$_GET['file'].".";
```

Comment:
Using value directly from $_GET is opening the door for code injection. These should be sanitized before being used.

Reply:
Done.